lib/verification-reminders.js

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var config = require('../config')
+var P = require('./promise')
+var reminderConfig = config.get('verificationReminders')
+
+var LOG_REMINDERS_CREATED = 'verification-reminders.created'
+var LOG_REMINDERS_DELETED = 'verification-reminders.deleted'
+var LOG_REMINDERS_ERROR_CREATE = 'verification-reminder.create'
+var LOG_REMINDERS_ERROR_DELETE = 'verification-reminder.delete'
+
+module.exports = function (log, db) {
+  /**
+   * shouldRemind
+   *
+   * Determines if we should create a reminder for this user to verify their account.
+   *
+   * @returns {boolean}
+   */
+  function shouldRemind() {
+    // random between 0 and 100, inclusive
+    var rand = Math.floor(Math.random() * (100 + 1))
```

Comment:
That's how it we get the sampling rate that will help with the slow roll out of the feature. We did it this way with metrics on the content server. Did you have something else in mind for this?